### DISCUSSION TITLE
How many years of financial statements is IB providing?

### DISCUSSION THREAD

**Discussion by jab2727** [quantbelt/ib_fundamental#13] [General]
Curious before I dive into working with the IB API, how many years back are you typically able to get financial statements for?

**Comment by gnzsnz** [comment]
5 quarters and 6 years for annual data.

right now IBKR service for report 'ReportsFinStatements' is down. don't know when or if it will be up again. in the past it has been down intermittently. see #12

**Reply by jab2727** [reply]
Ok got it, I appreciate the reply!
